package.json

```diff
@@ -82,6 +82,7 @@
     "mysql2": "3.14.1",
     "node-jose": "2.2.0",
     "sanitize-html": "2.16.0",
+    "sharp": "^0.34.1",
```

Comment:
Lets pin this to a `0.34.1`

Reply:
Good catch, added `yarn config set save-prefix ""` to my `~/.yarnrc` config to avoid it in the future

Fixed in 25f417ef23b64c929a63287c9c516e3d02554012